src/ucp/core/ucp_request.h

```diff
@@ -60,6 +61,7 @@ typedef struct ucp_send_state {
  * Request in progress.
  */
 struct ucp_request {
+    ucs_status_t                  status;  /* Operation status */
```

Comment:
it's 176 bytes now as it was before (due to alignment/padding). Have not investigated the data struct layout with tools lake pahole due to that.
If we add status to the end, it becomes somewhat 182 or 184